tests/language-server-simulator/src/main/java/org/ballerinalang/langserver/simulator/generators/Generators.java

```diff
@@ -42,7 +42,7 @@ private Generators() {
         // Get skipped generators
         String property = System.getProperty(PROP_SKIPPED_GENERATORS, "");
         Set<Type> skippedGenerators = Stream.of(property.split(","))
-                .map(String::trim)
+                .filter(String::isBlank)
```

Comment:
We should add the negation of this condition ryt?

Reply:
Fixed